docs/v0.0.1/search-items.md

```diff
@@ -0,0 +1,41 @@
+# Search Result Items
+Search view consumes a flat list of search result items (in `json` format) for its contents.
+
+## Search item JSON format
+This flat list of search result items takes the following form: it must contain exactly one `searchItems` root item that represents a list of search result items. Each of the search result items is made up of the following fields: 
+
+- `text` - The name of a given search result item.
+- `iconUrl`- The absolute URL of icon of a given search result item.
+- `contextData` - The string value that will be passed to the event handler when a given search result item is clicked.
+- `itemType` - This value can either be `creation`, `action` or `query`. For more details of these values, please refer to [Layout Specification](./layout-specs.md) document.
+- `group` - The name of the group that a given search result item belongs to.
```

Comment:
I remember recommending against having `group` in my previous review, do you have a different opinion on that?